app/code/Ometria/Api/Controller/V2/Products.php

```diff
@@ -85,6 +85,9 @@ class Products extends Action
     /** @var AppEmulation */
     private $appEmulation;
 
+    /** @var InventoryService */
+    private $inventoryService;
+
```

Comment:
why this is included in Products API V2 only ?

Reply:
@mkuzdowicz this is used only in the Products V2 API as compatibility with Multi Stock Inventory was introduced therein. The inventory service has been added as a wrapper to try to retrieve the stock values regardless of the stock configuration (MSI or non-MSI).